docs/specs/toc.yml

```diff
@@ -850,6 +850,29 @@ outputs:
   docs/f1/TOC.experimental.json: |
     {"items":[{"toc_title":"Index Reference","href":"../"}]}
 ---
+# Paired experimental TOCs' metadata
+inputs:
```

Comment:
also add case for yaml format?

Reply:
you mean the experimental metadata for yaml? sure. I thought the yaml/json toc spec test has already covered that